lib/web_ui/lib/src/engine/host_node.dart

```diff
@@ -152,6 +155,18 @@ class ShadowDomHostNode implements HostNode {
 class ElementHostNode implements HostNode {
```

Comment:
This only applies in the case where ShadowDOM is not supported.

Reply:
I think with Stadia gone, we no longer have customers for shadow-DOMless mode.

Reply:
I thought the same too, but I wasn't sure that the number of customers for this would be exactly zero.

Reply:
I created this issue: https://github.com/flutter/flutter/issues/116204 so this can be tackled later.